src/popgun.scss

```diff
@@ -190,10 +190,12 @@ $boxShadowPop: 1px 2px 2px rgba(#1a333d, 0.15);
   overflow: visible;
   white-space: normal;
 
-  transition: top .25s ease-out, left .25s ease-out;
+  -webkit-transition: top .25s ease-in-out, left .25s ease-in-out;
+  transition: top .25s ease-in-out, left .25s ease-in-out;
```

Comment:
Uhh that was left over from when I was just playing around with the stylesheet.. good catch! I'll remove it! 😄